ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1148,12 +1151,45 @@ open class TableViewCell: UITableViewCell, TokenizedControlInternal {
 
     private var isUsingCustomTextColors: Bool = false
```

Comment:
Probably outside the scope of this PR, but would it be possible to remove this and instead just also use the attributed strings internally if someone tries to set a custom color? Not sure it's entirely possible, but it might be possible to make a new attributed string that has all of the old attributes (which could be none) plus the new custom color.

Reply:
I think it's worth the try. But I believe it warrants its own PR. The Label control in the experiment might also be involved because of this:
https://github.com/microsoft/fluentui-apple/blob/1e8d6806062c12aee072caff2b10cf8254e9a51b/ios/FluentUI/Label/Label.swift#L114

@sople could you log a work item that so we can remove the isUsingCustomTextColors var and experiment using NSAttributedStrings even when only the text is set?

Reply:
Surprised that someone else out there has my alias 😆 but yes will log the work item and experiment!

Reply:
Just edited the comment tagging the right one! 
:|